### PR TITLE
Add ttml generator

### PR DESCRIPTION
**Pull request by MarcAntoine-Arnaud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Worker to transcode media into DASH
 Dependencies
 ------------
 
-Depends on [py_amqp_connection](https://github.com/FTV-Subtil/py_amqp_connection) package:
+To depvelop locally, add dependencies with this command:
```
Comment (valnoel) [inline README.md]: _To __develop__ ..._

Comment (valnoel) [reply README.md]: _add __the__ dependencies_

File: gpac_worker/worker.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        try:
+            kind = msg['parameters']['kind']
+            if kind == "generate_dash":
+                generate_dash.process(conn, msg)
+            elif kind == "ttml_to_mp4":
+                ttml_to_mp4.process(conn, msg)
+            else:
+                error_content = {
+                    "body": body.decode('utf-8'),
+                    "error": "unable to process message, kind in unsupported",
```
Comment (valnoel) [inline gpac_worker/worker.py]: _kind __is not__ supported_
And add the `kind` value to the message?

File: gpac_worker/generate_dash.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+from gpac_worker import GPAC_worker
+
+def process(conn, msg):
+    src_paths = msg['parameters']['source']['paths']
+    if len(src_paths) == 0:
+        raise RuntimeError("No source specified")
+    # for path in src_paths:
+    #     if not os.path.exists(path):
+    #         logging.error("Source path '%s' is not reachable or does not exists.", path)
+    #         return False
```
Comment (valnoel) [inline gpac_worker/generate_dash.py:12]: You commented this because of the path suffixes? Shouldn't we find a way to handle them?

Comment (MarcAntoine-Arnaud) [reply gpac_worker/generate_dash.py:12]: Yes, and yes. But it require an issue for that.

**Pull request merged**